tools/cli.js

```diff
@@ -251,15 +272,22 @@ lineProcessor.connect = (tokens, callback) => {
     return callback(reportMissingArgument('Host'));
   }
   const args = _split(tokens, ' ', 2);
-  const [host, port] = _split(args[0], ':');
+  let [scheme, authority] = _split(args[0], '://', 1, true);
+  if (authority === undefined) {
```

Comment:
With this command `connect ://host.com app` we get wrong error `Error occurred: Host is not provided`